rskj-core/src/main/java/co/rsk/net/sync/DownloadingBackwardsBodiesSyncState.java

```diff
@@ -98,6 +102,8 @@ public void newBody(BodyResponseMessage body, MessageChannel peer) {
 
         if (child.getNumber() == 1) {
             connectGenesis(child);
+            blockStore.flush();
```

Comment:
Nice fix, i'd consider moving it to another commit

Reply:
I'll move the changes on this class to another commit :+1: